core-services/release-controller/_releases/release-ocp-4.12.json

```diff
@@ -70,7 +70,7 @@
       "maxRetries": 3,
       "optional": true,
       "prowJob": {
-        "name": "periodic-ci-openshift-release-master-nightly-4.12-e2e-aws"
+        "name": "periodic-ci-openshift-release-master-nightly-4.12-e2e-aws-sdn"
```

Comment:
Shouldn't we rename this to aws-sdn above on line 69?

Reply:
yes. good catch. fixed.